core/primitives/address.cpp

```diff
@@ -4,7 +4,170 @@
  */
 
 #include "primitives/address.hpp"
```

Comment:
add empty line after this include. (Check google code style for the order of includes)